### ISSUE TITLE
Expansion Panel Header gets hidden

### ISSUE THREAD

**Issue by mubashshir-uneecops** [angular/components#16879]
This problem is raised by the testing team. header gets hidden when they try to find some text by selenium xpath. It should be highlighted instead of getting hidden.

Before - 
![image](https://user-images.githubusercontent.com/50190870/63684920-b9e55f00-c81b-11e9-9633-aeda63ede6c4.png)

After i search 'textToFind' from developer console by using the command -
//span[contains(text(),'textToFind')]
then expansion panel header gets hidden

![image](https://user-images.githubusercontent.com/50190870/63684987-e305ef80-c81b-11e9-822f-fc33c5b1f790.png)

**Comment by tahaabu** [comment]
Can you share the stackbBlitz code link ?

**Comment by mubashshir-uneecops** [comment]
Yes, Sure
https://stackblitz.com/edit/angular-czakrt

**Comment by mubashshir-uneecops** [comment]
Can anyone help me with the solution. Testing team is sitting idle waiting for it to get resolved.
I have heavily used expansion panel in my project

**Comment by josephperrott** [comment]
It looks like this is actually occurring because when the browser "finds" the text in the expansion panel body, it pushes the expansion panel header up.  Since the expansion panel has `overflow: hidden`, the header text is "above" the expansion panel header container, and the expansion panel body is within the header container, but `visibilty: hidden` is still applied to it.

**Comment by jelbourn** [comment]
Closing this since I believe that this isn't a bug that would affect human use of the component, and the official test harnesses can be used to interact with the component in tests
